views/js/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -365,7 +365,7 @@ define([
         var $container = containerHelper.get(interaction);
         var attributes = interaction.getAttributes();
         var responseDeclaration = interaction.getResponseDeclaration();
-        var baseType = responseDeclaration.attr('baseType');
+        var baseType = responseDeclaration ? responseDeclaration.attr('baseType') : '';
```

Comment:
Thank you @krampstudio I will see your PR, but I work on it now, so have not updated this PR yet.